pkg/process/config/config.go

```diff
@@ -493,6 +498,11 @@ func loadSysProbeEnvVariables() {
 		{"DD_DISABLE_DNS_INSPECTION", "system_probe_config.disable_dns_inspection"},
 		{"DD_COLLECT_LOCAL_DNS", "system_probe_config.collect_local_dns"},
 		{"DD_COLLECT_DNS_STATS", "system_probe_config.collect_dns_stats"},
+		{"DD_SYSTEM_PROBE_PROFILING_ENABLED", "system_probe_config.profiling.enabled"},
+		{"DD_SITE", "system_probe_config.profiling.site"},
+		{"DD_APM_PROFILING_DD_URL", "system_probe_config.profiling.profile_dd_url"},
+		{"DD_API_KEY", "system_probe_config.profiling.api_key"},
+		{"DD_ENV", "system_probe_config.profiling.env"},
```

Comment:
I think all of these should have the `DD_PROFILING_` prefix.

Reply:
Or actually `DD_SYSTEM_PROBE_PROFILING_`

Reply:
The idea here was to reuse some of the existing env var names, so it wasn't confusing.

Reply:
These values don't need to differ from the main agent. We simply don't have access to the main agent config **file** from the system-probe, and it complicated helm charts too much to add. Using env vars is an easy way to be able to pass the same values, without needing additional fs mounts.

Reply:
Oh, you're just reusing some of the existing env vars. Carry on :-)